slick.grid.css.map

```diff
@@ -1,7 +1,7 @@
+"mappings": "AAkBA;;;gBAAmB;EAIjB,UAAU,EAAE,UAAU;;AAExB,sBAAsB;EACpB,OAAO,EAAE,eAAe;;AAG1B,kBAAkB;EAEhB,MAAM,EADF,IAAI;EAER,wBAAK;IAEH,OAAO,EADP,GAAG;IAEH,WAAW,EAAE,IAAe;;AAEhC,qBAAqB;EAEnB,MAAM,EADF,IAAI;EAER,2BAAK;IAEH,OAAO,EADP,GAAG;IAEH,WAAW,EAAE,IAAe;;AAGhC,eAAe;EAEb,MAAM,EADF,IAAI;EAER,WAAW,EAFP,IAAI;EAGR,qBAAK;IACH,OAAO,EAAE,KAAK;;AAKlB,UAAU;EACR,QAAQ,EAAE,MAAM;EAChB,OAAO,EAAE,CAAC;EAEV,oBAAS;IACP,UAAU,EAAE,OAAO;EAErB,sBAAW;IACT,QAAQ,EAAE,KAAK;IACf,KAAK,EAAE,CAAC;IACR,MAAM,EAAE,CAAC;IACT,GAAG,EAAE,CAAC;IACN,IAAI,EAAE,CAAC;IACP,OAAO,EAAE,CAAC;EAEZ;sBAAe;IAEb,QAAQ,EAAE,QAAQ;IAClB,KAAK,EAAE,IAAI;EAEb,yBAAc;IACZ,MAAM,EAAE,CAAC;IACT,QAAQ,EAAE,MAAM;IAEhB,+CAAqB;MACnB,OAAO,EAAE,CAAC;EAEd,oBAAS;IACP,QAAQ,EAAE,IAAI;IACd,OAAO,EAAE,CAAC;IAKV,+EAAM;MAGJ,QAAQ,EAAE,MAAM;IAElB,qCAAkB;MAChB,MAAM,EAAE,IAAI;MACZ,IAAI,EAAE,CAAC;MACP,GAAG,EAAE,CAAC;EAEV,kBAAO;IACL,QAAQ,EAAE,QAAQ;EAEpB,eAAI;IACF,QAAQ,EAAE,QAAQ;IAClB,MAAM,EAAE,CAAC;IACT,KAAK,EAAE,IAAI;EAEb,gBAAK;IACH,QAAQ,EAAE,QAAQ;IAClB,MAAM,EAAE,IAAI;IACZ,MAAM,EAAE,qBAAqB;IAC7B,YAAY,EAAE,iBAAiB;IAC/B,mBAAmB,EAAE,MAAM;IAC3B,QAAQ,EAAE,MAAM;IAChB,gBAAgB,EAAE,QAAQ;IAC1B,aAAa,EAAE,QAAQ;IACvB,cAAc,EAAE,MAAM;IACtB,OAAO,EAAE,CAAC;IACV,MAAM,EAAE,CAAC;IACT,WAAW,EAAE,MAAM;IACnB,MAAM,EAAE,OAAO;IACf,4BAAa;MACX,UAAU,EAAE,YAAY;MACxB,UAAU,EAAE,oBAAoB;MAChC,kBAAkB,EAAE,QAAQ;MAC5B,eAAe,EAAE,QAAQ;MACzB,aAAa,EAAE,QAAQ;MACvB,UAAU,EAAE,QAAQ;IACtB,yBAAU;MACR,MAAM,EAAE,wBAAwB;IAClC,yBAAU;MACR,OAAO,EAAE,EAAE;MACX,QAAQ,EAAE,OAAO;MACjB,UAAU,EAAE,KAAK;MACjB,YAAY,EAAE,KAAK;MACnB,YAAY,EAAE,KAAK;IACrB,sBAAO;MACL,OAAO,EAAE,IAAI;IACf,yBAAU;MACR,UAAU,EAAE,KAAK;IACnB,uBAAQ;MACN,UAAU,EAAE,YAAY;;AAE9B;qBAAmB;EAEjB,QAAQ,EAAE,QAAQ;EAClB,WAAW,EAAE,MAAM;EACnB,MAAM,EAAE,OAAO;EACf,QAAQ,EAAE,MAAM;EAChB,aAAa,EAAE,gBAAgB;EAC/B,UAAU,EAAE,OAAO;;AAInB;2BAAK;EACH,YAAY,EAAE,gBAAgB;EAC9B,WAAW,EAAE,CAAC;EACd,UAAU,EAAE,CAAC;EACb,aAAa,EAAE,CAAC;EAEhB;oCAAQ;IACN,UAAU,EAAE,WAAW;;AAE7B,qBAAqB;EACnB,UAAU,EAAE,IAAI;EAChB,iCAAW;IACT,UAAU,EAAE,UAAU;IACtB,KAAK,EAAE,IAAI;IACX,MAAM,EAAE,IAAI;;AAKhB,2BAA2B;EACzB,gBAAgB,EAAE,OAAO;;AAC3B,kCAAkC;EAChC,gBAAgB,EAAE,OAAO;;AAI3B,qBAAqB;EACnB,OAAO,EAAE,YAAY;EACrB,KAAK,EAAE,GAAG;EACV,MAAM,EAAE,GAAG;EACX,WAAW,EAAE,GAAG;EAChB,UAAU,EAAE,GAAG;EACf,KAAK,EAAE,IAAI;;AAEb,0BAA0B;EACxB,UAAU,EAAE,yBAAyB;;AAEvC,yBAAyB;EACvB,UAAU,EAAE,wBAAwB;;AAEtC,mBAAmB;EACjB,QAAQ,EAAE,QAAQ;EAClB,SAAS,EAAE,KAAK;EAChB,OAAO,EAAE,KAAK;EACd,MAAM,EAAE,UAAU;EAClB,KAAK,EAAE,GAAG;EACV,KAAK,EAAE,GAAG;EACV,GAAG,EAAE,CAAC;EACN,MAAM,EAAE,IAAI;;AAEd,2BAA2B;EACzB,UAAU,EAAE,MAAM;;AAMpB,mBAAmB;EACjB,OAAO,EAAE,YAAY;;AAIvB,oBAAoB;EAClB,OAAO,EAAE,YAAY;EACrB,UAAU,EAAE,IAAI;EAChB,OAAO,EAAE,IAAI;EACb,MAAM,EAAE,iBAAmB;EAC3B,MAAM,EAAE,IAAI;;AAEd,oBAAoB;EAClB,OAAO,EAAE,YAAY;EACrB,MAAM,EAAE,GAAG;EACX,UAAU,EAAE,IAAI;EAChB,OAAO,EAAE,GAAG;EACZ,MAAM,EAAE,iBAAmB;;AAE7B,gBAAgB;EACd,OAAO,EAAE,EAAE;EACX,QAAQ,EAAE,QAAQ;EAClB,MAAM,EAAE,gBAAgB",
```

Comment:
agreed. css might make sense to keep since it's the preferred consumable. map could be helpful for debugging.